crates/fuel-core/src/query/subscriptions/test.rs

```diff
@@ -1,3 +1,6 @@
+// Allow `arc_with_non_send_sync` to enable use of `Arbitrary`
+#![allow(clippy::arc_with_non_send_sync)]
```

Comment:
`arc_with_non_send_sync` is a new Clippy lint that was introduced in Rust 1.72.0 (see https://rust-lang.github.io/rust-clippy/master/index.html#/arc_with_non_send_sync). Currently, structs that derive `Arbitrary` fail this lint due to its dependency on `Proptest`, which in turn uses `Arc`.

Reply:
Looks like this was actually fixed, and we just need the update to propagate upstream: https://github.com/proptest-rs/proptest/issues/364